net/sslh/src/opnsense/mvc/app/models/OPNsense/Sslh/Settings.xml

```diff
@@ -0,0 +1,66 @@
+<model>
+    <mount>//OPNsense/sslh</mount>
+    <version>1.21c.1</version>
```

Comment:
```suggestion
    <version>0.0.1</version>
```

model versioning should also be different from software version. can be linked to plugin version, but most of the time it isn't and it doesn't matter as long as version is increased when necessary (changed defaults, new required values, removed items).